improver/calibration/reliability_calibration.py

```diff
@@ -462,27 +441,31 @@ def __repr__(self):
         """Represent the configured plugin instance as a string."""
         return '<AggregateReliabilityCalibrationTables>'
 
-    def _construct_single_cube(self, cubes):
+    @staticmethod
```

Comment:
I think that the `self.cube_index_coord` and `self.merge_coord` can now be removed from the `__init__`.

Reply:
Done.